src/store/reducers/user-reducer.js

```diff
@@ -1,57 +1,36 @@
 import { userService } from "../../services/user";
 
-export const INCREMENT = "INCREMENT";
-export const DECREMENT = "DECREMENT";
-export const CHANGE_COUNT = "CHANGE_COUNT";
+export const SET_USERS = "SET_USERS";
 export const SET_USER = "SET_USER";
 export const SET_WATCHED_USER = "SET_WATCHED_USER";
 export const REMOVE_USER = "REMOVE_USER";
```

Comment:
I agree with the term **`delete`** rather than `remove` for user, as this means permanent elimination of the user from the storage rather than disassociating him (for example, in MongoDB we use the `db.collection.deleteOne(filter)` method to delete a single document matching the filter from the db collection, so I think this also reinforces the use of the term **`delete`** over `remove`).

So instead of `REMOVE_USER` I think we should use **`DELETE_USER`** and use here in the term **`delete`** anywhere within `user` scope.